### DISCUSSION TITLE
Apply WithoutValidation attribute to all properties in a class.

### DISCUSSION THREAD

**Discussion by quyle92** [spatie/laravel-data#210] [Q&A]
Hi,
Is there any way that I can apply the `WithoutValidation` attribute to all properties in my Data class.
Here is the current:
```
class SongData extends Data
{
    public function __construct(
        #[WithoutValidation] //have to manually specify this on each prop.
        public string $name,
        #[WithoutValidation]
        public string $lyrics,
    ) {
    }
}
```
Any idea on how to not doing this repetitively for each props  would be much appreciated.

**Comment by Nir-An** [comment]
Hi @quyle92 
I think it should help but I didn't test it 
https://spatie.be/docs/laravel-data/v2/as-a-data-transfer-object/request-to-data-object#content-skipping-validation


```php
    public static function fromRequest(Request $request): static{
        return new self("{$request->input('name')} {$request->input('lyrics')}")
    }

```

**Reply by quyle92** [reply]
thanks but it still needs to use `WithoutValidation` for each prop.

**Comment by rubenvanassche** [comment]
I'm open if someone Pr's it! Don't need it at the moment so I sadly have no time to take a look at it.
